### ISSUE TITLE
Windows High Contrast Mode Issues for 526 Review page

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95550]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-1
**Page**:  review-and-submit
**Environment**: Windows High Contrast Mode
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Not sure
**Relevant WCAG success criteria**: Best practice

### To Reproduce:
- https://devtoolstips.org/tips/en/emulate-forced-colors/

Issues in WHCM for the following items: 
- The breadcrumb nav's > icon is not visible
![Image](https://github.com/user-attachments/assets/1e0824ce-4f01-4f04-9880-db3e25a8409d)

- Missing borders on buttons such as Back / Submit application / Save / Cancel / Feedback / Update page
![Image](https://github.com/user-attachments/assets/c6a9e69c-5d12-4e27-a31b-4f5b80b5d621)

- The loading indicator appears as a plain circle, not a spinner.
- Cannot see checkbox when it is not focused in WHCM
![Image](https://github.com/user-attachments/assets/35b83431-8939-46ca-a031-5dd51114bb42)

- Cannot tell which radio button is checked in WHCM
![Image](https://github.com/user-attachments/assets/b12e1c17-1554-44f1-abb3-5a156da5e562)

- The Progress Stepper and circle around the 5 are not visible in WHCM
![Image](https://github.com/user-attachments/assets/1e51a6e9-7b0d-4779-b447-4c07500241a8)

- Periods of confinement up and down arrows, border missing on update button / add another period button
![Image](https://github.com/user-attachments/assets/75b2b9b6-87ac-4307-aedd-cb793346d6ba)

## Tasks
- [ ] Make items WHCM friendly by adding transparent borders or by using `currentColor` for SVGs
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Images & borders should be visible in Windows High Contrast Mode
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
